### PR TITLE
feat: finish first couple screens of Intro

### LINKED ISSUE #10: 🚀 Feature: Set up transitions between screens (with movement of rocket)
Using the [anime.js](https://animejs.com/) library will make this really easy.

* [ ] :wave: I may be able to implement this feature request

---

This is a :rocket: Feature Request

### PR DESCRIPTION
**Pull request by BryanPan342**
* animations with anime.js for the first couple of screens

Fixes #10

### CODE REVIEW

File: src/components/Home/Intro.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    .add({
+      targets: '#rocket-path',
+      width: ['857px', '1606px'],
+      duration: 2500,
+      easing: 'easeInOutExpo',
+    }, '-=3000');
+  }, []);
+
+  useEffect(() => {
+    timeline.current?.seek(slideIdx * 2000);
```
Comment (reginawang99) [inline src/components/Home/Intro.tsx:114]: can we do a clearTimeout at the beginning of this, like `clearTimeout(timer);`? the next and back slides sometimes wonk out

**Review by reginawang99 — CHANGES_REQUESTED** [review]
looks good to me! sorry just a small like animation thing, i can also do it in another commit if you dont want to change it! :)

File: src/components/Home/Intro.tsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        <div id={'rocket-path'}/>
+        <svg id={'bg-trajectory'} width="1606" height="128" viewBox="0 0 1606 128" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path id={'path-1'} d="M4 4C4 4 81 4 339.5 4" stroke="black" strokeWidth="8" strokeLinecap="round" strokeDasharray="20 30"/>
+          <path id={'path-2'} d="M329 4C447 4 511 18.5 511 61.5C511 104.5 594 123.5 852.5 123.5" stroke="black" strokeWidth="8" strokeLinecap="round" strokeDasharray="20 30"/>
+          <path id={'path-3'} d="M843 124C843 124 1309.38 124 1602 124" stroke="black" strokeWidth="8" strokeLinecap="round" strokeDasharray="20 30"/>
+        </svg>
+        <div id={'rocket'}/>
+      </div>
+      <div id={'server-container'}>
+        <div id={'library'}/>
```
[the remark this answers is not in the document]
Comment (BryanPan342) [reply src/components/Home/Intro.tsx:133]: ooooo

File: src/components/Home/Intro.tsx (hunk 1 above)
Comment (reginawang99) [reply src/components/Home/Intro.tsx:114]: hmm actually that might require a timer state variable so idk if thats the best way

File: src/components/Home/Intro.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (BryanPan342) [reply src/components/Home/Intro.tsx:133]: okay because i was a tad confused on where it was gonna go

File: src/components/Home/Intro.tsx (hunk 1 above)
Comment (BryanPan342) [reply src/components/Home/Intro.tsx:114]: ah yes thats smart

i got around this by setting up a delay

File: src/components/Home/Intro.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (BryanPan342) [reply src/components/Home/Intro.tsx:133]: wait so.. server zooms out and then get replaced by servers?

[the remark this answers is not in the document]
Comment (reginawang99) [reply src/components/Home/Intro.tsx:133]: yuhhh :^) server zooms out to servers (maybe we can do fade in if thats easier)

**Review by reginawang99 — APPROVED** [review]
noiceee looks good to me!! the refs stuffs is cool hehe and also anime so fancy :)))))

**Pull request merged**